src/libs/widget-lib/src/cowSwapWidget.ts

```diff
@@ -1,15 +1,15 @@
-import { DEFAULT_HEIGHT, DEFAULT_WIDTH } from './consts'
 import { JsonRpcManager } from './JsonRpcManager'
-import { CowSwapWidgetParams, CowSwapWidgetUrlParams } from './types'
-import { buildTradeAmountsQuery, buildWidgetPath, buildWidgetUrl } from './utils'
+import { CowSwapWidgetParams, CowSwapWidgetSettings } from './types'
+import { buildTradeAmountsQuery, buildWidgetPath, buildWidgetUrl } from './urlUtils'
 
-const COW_SWAP_WIDGET_KEY = '@cowprotocol/widget-lib'
+const COW_SWAP_WIDGET_URL_UPDATE_KEY = 'cowSwapWidgetUrlUpdate'
+const COW_SWAP_WIDGET_APP_UPDATE_KEY = 'cowSwapWidgetAppUpdate'
```

Comment:
Why not having one key for all messages comming from the widget, like `cowswap-widget` or whatever, and then we add a `method` property, and we list the possible values there: i.e. `url-update` or `app-update`?

Reply:
Makes sense, will do that, thanks!

Reply:
Fixed